index.html

```diff
@@ -0,0 +1,552 @@
+                        <p class="link-pic-title">Jeff Bezos' Blue Origin to Partner With 3 Companies on NASA Moon Lander</p>
+                        <p class="link-date">Oct. 22</p>
+                    </div>
+                    <div class="center more-wrapper">
+                        <img src="https://static01.nyt.com/images/2019/12/03/science/26JUPITER1/26JUPITER1-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="nasa women">
+                        <p class="link-pic-descr">NASA, via Associated Press</p>
+                        <p class="link-pic-title">Why NASA's First All-Women Spacewalk Made History</p>
+                        <p class="link-date">Oct. 18</p>
+                    </div>
+                    <div class="right more-wrapper">
+                        <img src="https://static01.nyt.com/images/2019/06/04/science/11starlink01/31SCI-SATELLITES2-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale"
+                            alt="ads">
+                        <div class="advert-small">ADVERTISEMENT</div>
+                    </div>
+                    <div class="left more-wrapper">
+                        <img src="https://static01.nyt.com/images/2019/11/26/science/20MARS/20MARS-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="aleksei leonov">
+                        <p class="link-pic-descr">Associated Press</p>
+                        <p class="link-pic-title">Aleksei Leonov, First to Walk in Space, Dies at 85</p>
+                        <p class="link-date">Oct. 11</p>
+                    </div>
+                    <div class="center more-wrapper">
+                        <img src="https://static01.nyt.com/images/2019/11/19/science/14BLACKHOLE1/14BLACKHOLE1-threeByTwoSmallAt2X.jpg?quality=75&auto=webp&disable=upscale" alt="">
+                        <p class="link-pic-descr">Jonathan Nackstrand/Agence France-Presse - Getty</p>
+                        <p class="link-pic-title">Nobel Prize in Physics Awarded for Studies of Earth's Place in the Universe
+                        </p>
+                        <p class="link-date">Oct. 8</p>
+                    </div>
+                    <div class="right more-wrapper">
+                        <img src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/42/Distant_galaxy_GN-z11_in_GOODS-N_image_by_HST.jpg/300px-Distant_galaxy_GN-z11_in_GOODS-N_image_by_HST.jpg"  alt="space paths for women">
+                        <p class="link-pic-descr">Alison Farrand for the New York Times</p>
+                        <p class="link-pic-title">An Astronaut Who Built Paths to Space for Other Women</p>
+                        <p class="link-date">Oct. 2</p>
+                    </div>
+                </div>
+            </div>
+            <aside class="most-popular-container">
+                <h4>Most Popular</h4>
+                <p><a>Ja Morant Is the N.B.A.’s Newest Phenom</a></p>
```

Comment:
This should be links, please add the `href` attribute so the work as expected. You can use a `#` for the `href`: `<a href="#">Some article</a>`.